.github/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 2. Bug - Fixed online ring display for ESSEX_APP (now shows 1x ring rather than 3x) - thanks to @hazzas-99
 3. AIRAC (2306) - Updated Humberside (EGNJ) Runway Headings - thanks to @AmanT0mar (Aman Tomar)
 4. AIRAC (2305) - Updated Various MIL RWY Headings - thanks to @DuanBoomer (Chirag)
+5. Enhancement - Updated Manchester (EGCC) Stand Positon - thanks to @rishab-alt (Rishab Saddiq(Rish)) 
```

Comment:
```suggestion
5. Bug - Corrected Manchester (EGCC) Stand 12 Positon - thanks to @rishab-alt (Rishab Saddiq(Rish)) 
```
I would suggest this is a bug fix as the stand was in the wrong position. See also https://github.com/VATSIM-UK/UK-Sector-File/pull/4756#discussion_r1210529240 regarding your name.